src/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -465,26 +471,85 @@ class OSRow extends React.Component {
     }
 }
 
-const OfflineTokenRow = ({ offlineToken, onValueChanged, validationFailed }) => {
-    const validationStateToken = validationFailed.offlineToken ? 'error' : 'default';
+// This method needs to be outside of component as re-render would create a new instance of debounce
+// Debounce will trigger "getAccessToken" only if >250ms has passed since user changed offlineToken
+// Since "getAccessToken" basically triggers HTTP request, this prevents user from triggering dozens of HTTP requests
+// while typing an offline token
+const getAccessTokenDebounce = debounce(500, (offlineToken, onValueChanged, setValidationState, validationStates) => {
+    getAccessToken(offlineToken)
+            .then(out => {
+                const accessToken = out.trim();
+                onValueChanged("accessToken", accessToken);
+                setValidationState(validationStates.SUCCESS);
+            })
+            .catch(ex => {
+                console.error(`Offline token validation failed: "${JSON.stringify(ex)}"`);
+                onValueChanged("accessToken", "");
+                setValidationState(validationStates.FAILED);
+            });
+});
+
+const OfflineTokenRow = ({ offlineToken, onValueChanged, formValidationFailed }) => {
+    const link = <a href="https://access.redhat.com/management/api" target="_blank" rel="noopener noreferrer">
+        <ExternalLinkAltIcon className="pf-u-mr-xs" />
+        {_("Get a new RHSM token.")}
+    </a>;
+    const validationStates = {
+        DEFAULT: {
+            option: "default",
+            message: <span>
+                { link }
+                {" " + _(" Then copy the token and paste it above.")}
```

Comment:
Why is there space concatenated with string starting with empty string? Translators are likely to break the leading space.